### PR TITLE
chore(storybook): move canary to experimental section

### LINKED ISSUE #4645: Move experimental / canary components to a separate section in Storybook
Realized during @paul-balchin-ibm’s FilterPanel PRs that the subcomponents were missing their Canary flags. After much frustration over this “black box” of why they were missing, realized that Storybook default sidebar width has them out of view.

A long subcomponent name shouldn’t remove all visibility into a component’s stability/status. One way we could address this is moving all experimental/unstable components into a separate section rather than within the top-level component/patterns.

| Sidebar default width | Sidebar resized |
------------------------|------------------
Image | Image

**This story focuses on moving the Canary and feature flags under the same section within Storybook.**
Other work to move the implementation of Canary to be more like feature flags is out of scope for this issue.

```
pkg.component.ActionBar = true; // this is the required step to enable a canary component for example
```

```[tasklist]
### Tasks
- [ ] Move “Canary” components and patterns to an Experimental section
- [ ] Update links for AVT tests
```

### PR DESCRIPTION
**Pull request by ariellalgilmore**
Closes #4645 

move necessary components to experimental section

#### What did you change?

all components labeled canary are now moved in storybook to experimental
also avt tests have been updated

#### How did you test and verify your work?
checked storybook and ran avt tests

### CODE REVIEW

**Comment by elycheea** [comment]
Let’s discuss this one in next Monday’s huddle so we can get some design input on this too.

**Review by oliviaflory — APPROVED** [review]
@ariellalgilmore this looks good to me! 

A few next steps we discussed in Huddle:
- Hide `Internal` components ?
- Remove `Community` section?
- It would be nice for Onboarding and Patterns to appear in the same order in both Production ready and Experimental sections. I would vote for Onboarding to be below Patterns even though that isn't alphabetical order
- We should move all the recategorized components and patterns in storybook to reflect the new IA (about modal stood out) but we can open a new issue for these things!

File: packages/ibm-products/src/components/Checklist/Checklist.stories.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const taskLists = [
 ];
 
 export default {
-  title: 'IBM Products/Onboarding/Checklist',
+  title: 'IBM Products/OnBoarding/Checklist',
```
Comment (elycheea) [inline packages/ibm-products/src/components/Checklist/Checklist.stories.jsx:63]:
```suggestion
  title: 'IBM Products/Onboarding/Checklist',
```

File: packages/ibm-products/src/components/InlineTip/InlineTip.stories.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const InlineTipAnimation = new URL(
 import DocsPage from './InlineTip.docs-page';
 
 export default {
-  title: 'IBM Products/Onboarding/Inline tip/InlineTip',
+  title: 'Experimental/OnBoarding/Inline tip/InlineTip',
```
Comment (elycheea) [inline packages/ibm-products/src/components/InlineTip/InlineTip.stories.jsx:24]:
```suggestion
  title: 'Experimental/Onboarding/Inline tip/InlineTip',
```

**Review by elycheea — COMMENTED** [review]
Some rouge `OnBoarding`s that should be quick to fix.

**Review by elycheea — DISMISSED** [review]

**Review by elycheea — APPROVED** [review]

**Review by annawen1 — APPROVED** [review]

**Pull request merged**